### DISCUSSION TITLE
An error is reported when the Python version is plumed

### DISCUSSION THREAD

**Discussion by s1lence-yxh** [plumed/plumed2#1055] [Q&A]
PLUMED: ERROR
PLUMED: I cannot understand line: OPES_METAD LABEL=opesmtd ARG=d1z,c1 PACE=500 BARRIER=50 TEMP=300.0 STATE_WFILE=STATE FILE=KERNELS
PLUMED: Maybe a missing space or a typo?
PLUMED: ################################################################################
PLUMED: 
PLUMED: (core/PlumedMain.cpp:823) void PLMD::PlumedMain::readInputWords(const std::vector<std::__cxx11::basic_string<char> >&)
PLUMED: ERROR
PLUMED: I cannot understand line: OPES_METAD LABEL=opesmtd ARG=d1z,c1 PACE=500 BARRIER=50 TEMP=300.0 STATE_WFILE=STATE FILE=KERNELS
PLUMED: Maybe a missing space or a typo?
PLUMED: ################################################################################
Traceback (most recent call last):
  File "/home/user/yxh/mace/NH/md.py", line 30, in <module>
    sys.calc =Plumed(calc=macemp,
              ^^^^^^^^^^^^^^^^^^^
  File "/home/user/.conda/envs/mace/lib/python3.12/site-packages/ase/calculators/plumed.py", line 154, in __init__
    self.plumed.cmd("readInputLine", line)
  File "plumed.pyx", line 263, in plumed.Plumed.cmd
  File "plumed.pyx", line 178, in plumed.Plumed.cmd_low_level
  File "plumed.pyx", line 161, in plumed.Plumed.raise_exception
plumed.PlumedError: 
(core/PlumedMain.cpp:823) void PLMD::PlumedMain::readInputWords(const std::vector<std::__cxx11::basic_string<char> >&)
ERROR
I cannot understand line: OPES_METAD LABEL=opesmtd ARG=d1z,c1 PACE=500 BARRIER=50 TEMP=300.0 STATE_WFILE=STATE FILE=KERNELS
Maybe a missing space or a typo?
PLUMED:                                               Cycles        Total      Average      Minimum      Maximum
PLUMED:                                                    1     0.002480     0.002480     0.002480     0.002480

please tell me what makes this mistake

**Comment by GiovanniBussi** [comment]
Did you compile plumed with opes module enabled?

Sent from Gmail mobile


Il giorno sab 30 mar 2024 alle 12:12 s1lence-yxh ***@***.***>
ha scritto:

**Reply by s1lence-yxh** [reply]
Thank you for the reply! I installed Plumed by Conda. How can I make opes module enabled? I could not find "./configure --enable-modules=opes". Is it "./config.sh --enable-modules=opes"?

**Comment by GiovanniBussi** [comment]
I think on conda forge this is not possible. You either have to:
- install plumed from source
- use the version that we provide with master classes (look for the
documentation in the 2022 series), hopefully it is stil functional

Sent from Gmail mobile


Il giorno dom 31 mar 2024 alle 07:03 s1lence-yxh ***@***.***>
ha scritto:

**Reply by s1lence-yxh** [reply]
Thank you very much. Sadly, I want to use Plumed by ASE. It is very unfortunate that I cannot use module of opes.
